packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -1687,23 +1687,33 @@ class _InputDecoratorState extends State<InputDecorator> with TickerProviderStat
         ),
       );
 
-    final Widget prefixIcon = decoration.prefixIcon == null ? null :
-      IconTheme.merge(
-        data: new IconThemeData(
-          color: iconColor,
-          size: iconSize,
+    Widget prefixIcon;
```

Comment:
I believe that the original syntax, 'final ... foo == null ? null : ..." is used consistently in this file.

Reply:
Reverted